pkg/gui/services/custom_commands/handler_creator.go

```diff
@@ -187,10 +187,13 @@ func (self *HandlerCreator) finalHandler(customCommand config.CustomCommand, ses
 		if customCommand.Stream {
 			cmdObj.StreamOutput()
 		}
-		err := cmdObj.Run()
+		output, err := cmdObj.RunWithOutput()
 		if err != nil {
 			return self.c.Error(err)
 		}
+		if customCommand.ShowOutput {
+			return self.c.Alert(cmdStr, output)
```

Comment:
Should we denote it in a special way if the output is `empty` so the user doesn't get confused? Or even omit the popup if the `output` is empty?

Reply:
Yeah, I think that there should be some kind of output, `<empty output>` maybe. No popup might confuse the user even more as to why `showOutput` is not working. I'll add it now.